src/components/Watchlists/NewWatchlistForm.js

```diff
@@ -0,0 +1,113 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import cx from 'classnames'
+import { Button, Toggle, Input } from '@santiment-network/ui'
+import * as actions from './../../actions/types'
+import styles from './NewWatchlistForm.module.scss'
+
+const MAX_NAME_LENGTH = 50
+const ERROR_MESSAGE =
+  'Something went wrong while creating watchlist, please try again!!!'
```

Comment:
you show this error, when input is empty. In design "Create" button should be disabled for this case and without error